### DISCUSSION TITLE
statics embedding layer with input size 0

### DISCUSSION THREAD

**Discussion by luc-henry** [neuralhydrology/neuralhydrology#31] [Q&A] [answered]
Hi, thanks for the package. 

Im trying your models with my own dataset and currently running into this issue:
`File "/home/ec2-user/SageMaker/neuralhydrology/neuralhydrology/modelzoo/inputlayer.py", line 77, in _get_embedding_net
    raise ValueError(f'Cannot create {purpose} embedding layer with input size 0')
ValueError: Cannot create statics embedding layer with input size 0`

I couldn't figure my mistake from the inputlayer.py

**Comment by kratzert** [comment, marked as answer]
Hi @henry-wni,
could you check in your config if you have settings defined for `statics_embedding`? If you don't use any static input features `static_attributes` or `evolving_attributes`, `statics_embedding` should be deleted or empty. Otherwise you are trying to create an embedding layer with zero static input features.

**Reply by kratzert** [reply]
Could you maybe sent your config then, so I could have a look?

**Comment by luc-henry** [comment]


**Reply by kratzert** [reply]
So in the conffig you sent me, you are defining a statics embedding and a dynamics embedding. Delete the entire passage of `statics_embedding` and try again to see if this is the problem.
![Screenshot from 2021-03-26 15-20-06](https://user-images.githubusercontent.com/13069767/112645508-d528ae00-8e46-11eb-99a2-61d89fc8b4fe.png)

**Reply by luc-henry** [reply]
my bad, thanks a lot.

**Reply by kratzert** [reply]
No problem, happy to help.
